apps/apple/FluentUITester/README.md

```diff
@@ -0,0 +1,33 @@
+# FluentUI React Native
+
+FluentUI React Native is a javascript component library that provides developers with controls that are part of the [Fluent Design System](https://www.microsoft.com/design/fluent/). These controls are built on [React Native](https://reactnative.dev/) and fully customizable.
+
+## Contributing
+
+This project welcomes contributions and suggestions. Most contributions require you to agree to a
+Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
+the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
+
+When you submit a pull request, a CLA bot will automatically determine whether you need to provide
+a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
+provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
+For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
+contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+### Developing in the repo
+
+Fluent UI Apple requires all [pull requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests) to come from forks of the repository. Please see [Fork a Repo - GitHub Help](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) for more details on how to set up a fork of microsoft/fluentui-react-native, keep it up-to-date, and submit pull requests back to this repository.
+
+FluentUI-React-Native takes on dependencies on [React-Native](https://github.com/facebook/react-native) for iOS, [React-Native](https://github.com/microsoft/react-native) for macOS, and [FluentUI-Apple](https://github.com/microsoft/fluentui-apple).
+
+After you've create a new fork to work in and installed the dependencies with yarn, you must perform some Apple specific steps before building.
+
+1. `cd apps/apple/FluentUITester`
+1. `yarn`
+1. `cd ios; pod install; cd ..`
+1. `cd macos; pod install; cd ..`
```

Comment:
These steps didn't work for me, even after making sure I had node 12 and the latest CocoaPods installed. For iOS I failed 
```
$ cd ios; pod install; cd .. 
[!] Invalid `Podfile` file: cannot load such file -- /Users/mavitale/Code/fluentui-react-native/apps/apple/FluentUITester/node_modules/@react-native-community/cli-platform-ios/native_modules. 


 #  from /Users/mavitale/Code/fluentui-react-native/apps/apple/FluentUITester/ios/Podfile:2 
 #  ------------------------------------------- 
 #  platform :ios, '12.0' 
 >  require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules' 
 #   
 #  ------------------------------------------- 
```

For macOS I got
```
$ cd macos; pod install; cd .. 
Analyzing dependencies 
Fetching podspec for `DoubleConversion` from `../node_modules/react-native-macos/third-party-podspecs/DoubleConversion.podspec` 


――― MARKDOWN TEMPLATE ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― 


### Command 


``` 
/usr/local/Cellar/cocoapods/1.9.1/libexec/bin/pod install 
``` 


### Report 


* What did you do? 


* What did you expect to happen? 


* What happened instead? 




### Stack 


``` 
   CocoaPods : 1.9.1 
        Ruby : ruby 2.6.3p62 (2019-04-16 revision 67580) [universal.x86_64-darwin19] 
    RubyGems : 3.0.3 
        Host : Mac OS X 10.15.4 (19E266) 
       Xcode : 11.4 (11E146) 
         Git : git version 2.24.1 (Apple Git-126) 
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib 
Repositories : master - git - https://github.com/CocoaPods/Specs.git @ 2cce9b94379894b7fc0a79d411a41395ae1d10be 


               visualstudio-defaultcollection-_git-cocoapods - git - https://msblox.visualstudio.com/defaultcollection/_git/cocoapods @ b57949d086529b5bb61e6116a862027a6acd3c7e 


``` 


### Plugins 


``` 
cocoapods-deintegrate : 1.0.4 
cocoapods-plugins     : 1.0.0 
cocoapods-search      : 1.0.0 
cocoapods-stats       : 1.1.0 
cocoapods-trunk       : 1.4.1 
cocoapods-try         : 1.1.0 
``` 


### Podfile 


```ruby 
# require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules' 


abstract_target 'Shared' do 
  # use_native_modules! 


  pod 'React', :path => "../node_modules/react-native-macos/" 
  pod 'React-Core', :path => "../node_modules/react-native-macos/React" 
  pod 'React-fishhook', :path => "../node_modules/react-native-macos/Libraries/fishhook" 
  pod 'React-RCTActionSheet', :path => "../node_modules/react-native-macos/Libraries/ActionSheetIOS" 
  pod 'React-RCTAnimation', :path => "../node_modules/react-native-macos/Libraries/NativeAnimation" 
  pod 'React-RCTBlob', :path => "../node_modules/react-native-macos/Libraries/Blob" 
  pod 'React-RCTImage', :path => "../node_modules/react-native-macos/Libraries/Image" 
  pod 'React-RCTLinking', :path => "../node_modules/react-native-macos/Libraries/LinkingIOS" 
  pod 'React-RCTNetwork', :path => "../node_modules/react-native-macos/Libraries/Network" 
  pod 'React-RCTSettings', :path => "../node_modules/react-native-macos/Libraries/Settings" 
  pod 'React-RCTText', :path => "../node_modules/react-native-macos/Libraries/Text" 
  pod 'React-RCTVibration', :path => "../node_modules/react-native-macos/Libraries/Vibration" 
  pod 'React-RCTWebSocket', :path => "../node_modules/react-native-macos/Libraries/WebSocket" 
  pod 'React-cxxreact', :path => "../node_modules/react-native-macos/ReactCommon/cxxreact" 
  pod 'React-jscallinvoker', :path => "../node_modules/react-native-macos/ReactCommon/jscallinvoker" 
  pod 'React-jsi', :path => "../node_modules/react-native-macos/ReactCommon/jsi" 
  pod 'React-jsiexecutor', :path => "../node_modules/react-native-macos/ReactCommon/jsiexecutor" 
  pod 'React-jsinspector', :path => "../node_modules/react-native-macos/ReactCommon/jsinspector" 
  pod 'yoga', :path => "../node_modules/react-native-macos/ReactCommon/yoga" 
  pod 'DoubleConversion', :podspec => "../node_modules/react-native-macos/third-party-podspecs/DoubleConversion.podspec" 
  pod 'glog', :podspec => "../node_modules/react-native-macos/third-party-podspecs/glog.podspec" 
  pod 'Folly', :podspec => "../node_modules/react-native-macos/third-party-podspecs/Folly.podspec" 
  pod 'boost-for-react-native', :podspec => "../node_modules/react-native-macos/third-party-podspecs/boost-for-react-native.podspec" 
  pod 'React-DevSupport', :path => "../node_modules/react-native-macos/React" 


  target 'FluentUITester-macOS' do 
    platform :macos, '10.15' 
    # Pods specifically for macOS target 
  end 


end 
``` 


### Error 


``` 
Errno::ENOENT - No such file or directory @ rb_sysopen - /Users/mavitale/Code/fluentui-react-native/apps/apple/FluentUITester/node_modules/react-native-macos/third-party-podspecs/DoubleConversion.podspec 
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/open-uri.rb:37:in `initialize' 
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/open-uri.rb:37:in `open' 
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/open-uri.rb:37:in `open' 
/usr/local/Cellar/cocoapods/1.9.1/libexec/gems/cocoapods-1.9.1/lib/cocoapods/external_sources/podspec_source.rb:19:in `block in fetch' 
/usr/local/Cellar/cocoapods/1.9.1/libexec/gems/cocoapods-1.9.1/lib/cocoapods/user_interface.rb:86:in `titled_section' 
/usr/local/Cellar/cocoapods/1.9.1/libexec/gems/cocoapods-1.9.1/lib/cocoapods/external_sources/podspec_source.rb:11:in `fetch' 
/usr/local/Cellar/cocoapods/1.9.1/libexec/gems/cocoapods-1.9.1/lib/cocoapods/installer/analyzer.rb:980:in `fetch_external_source' 
/usr/local/Cellar/cocoapods/1.9.1/libexec/gems/cocoapods-1.9.1/lib/cocoapods/installer/analyzer.rb:959:in `block (2 levels) in fetch_external_sources' 
/usr/local/Cellar/cocoapods/1.9.1/libexec/gems/cocoapods-1.9.1/lib/cocoapods/installer/analyzer.rb:958:in `each' 
/usr/local/Cellar/cocoapods/1.9.1/libexec/gems/cocoapods-1.9.1/lib/cocoapods/installer/analyzer.rb:958:in `block in fetch_external_sources' 
/usr/local/Cellar/cocoapods/1.9.1/libexec/gems/cocoapods-1.9.1/lib/cocoapods/user_interface.rb:64:in `section' 
/usr/local/Cellar/cocoapods/1.9.1/libexec/gems/cocoapods-1.9.1/lib/cocoapods/installer/analyzer.rb:957:in `fetch_external_sources' 
/usr/local/Cellar/cocoapods/1.9.1/libexec/gems/cocoapods-1.9.1/lib/cocoapods/installer/analyzer.rb:117:in `analyze' 
/usr/local/Cellar/cocoapods/1.9.1/libexec/gems/cocoapods-1.9.1/lib/cocoapods/installer.rb:410:in `analyze' 
/usr/local/Cellar/cocoapods/1.9.1/libexec/gems/cocoapods-1.9.1/lib/cocoapods/installer.rb:235:in `block in resolve_dependencies' 
/usr/local/Cellar/cocoapods/1.9.1/libexec/gems/cocoapods-1.9.1/lib/cocoapods/user_interface.rb:64:in `section' 
/usr/local/Cellar/cocoapods/1.9.1/libexec/gems/cocoapods-1.9.1/lib/cocoapods/installer.rb:234:in `resolve_dependencies' 
/usr/local/Cellar/cocoapods/1.9.1/libexec/gems/cocoapods-1.9.1/lib/cocoapods/installer.rb:156:in `install!' 
/usr/local/Cellar/cocoapods/1.9.1/libexec/gems/cocoapods-1.9.1/lib/cocoapods/command/install.rb:52:in `run' 
/usr/local/Cellar/cocoapods/1.9.1/libexec/gems/claide-1.0.3/lib/claide/command.rb:334:in `run' 
/usr/local/Cellar/cocoapods/1.9.1/libexec/gems/cocoapods-1.9.1/lib/cocoapods/command.rb:52:in `run' 
/usr/local/Cellar/cocoapods/1.9.1/libexec/gems/cocoapods-1.9.1/bin/pod:55:in `<top (required)>' 
/usr/local/Cellar/cocoapods/1.9.1/libexec/bin/pod:23:in `load' 
/usr/local/Cellar/cocoapods/1.9.1/libexec/bin/pod:23:in `<main>' 
``` 


――― TEMPLATE END ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― 


[!] Oh no, an error occurred. 


Search for existing GitHub issues similar to yours: 
https://github.com/CocoaPods/CocoaPods/search?q=No+such+file+or+directory+%40+rb_sysopen+-+%2FUsers%2Fmavitale%2FCode%2Ffluentui-react-native%2Fapps%2Fapple%2FFluentUITester%2Fnode_modules%2Freact-native-macos%2Fthird-party-podspecs%2FDoubleConversion.podspec&type=Issues 


If none exists, create a ticket, with the template displayed above, on: 
https://github.com/CocoaPods/CocoaPods/issues/new 


Be sure to first read the contributing guide for details on how to properly submit a ticket: 
https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md 


Don't forget to anonymize any private data! 


Looking for related issues on cocoapods/cocoapods... 
Found no similar issues. To create a new issue, please visit: 
https://github.com/cocoapods/cocoapods/issues/new 
```

Note, I had followed the main readme instructions first before following these instructions.